common/src/test/java/io/seata/common/util/DurationUtilTest.java

```diff
@@ -0,0 +1,37 @@
+package io.seata.common.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class DurationUtilTest {
+
+    @Test
+    public void testParse() {
+        Assertions.assertNull(DurationUtil.parse("d"));
+        Assertions.assertNull(DurationUtil.parse("h"));
+        Assertions.assertNull(DurationUtil.parse("m"));
+        Assertions.assertNull(DurationUtil.parse("s"));
+        Assertions.assertNull(DurationUtil.parse("ms"));
+
+        Assertions.assertEquals("PT8H", DurationUtil.parse("8h").toString());
+        Assertions.assertEquals("PT8M", DurationUtil.parse("8m").toString());
+        Assertions.assertEquals("PT8S", DurationUtil.parse("8s").toString());
+        Assertions.assertEquals("PT192H", DurationUtil.parse("8d").toString());
```

Comment:
I think it would probably be preferable to use `getSeconds()` here instead of `toString()` (in case `toString()` changes e.g. with system language) but it's probably not essential.

Reply:
Done